products/render-product.js

```diff
@@ -15,10 +12,10 @@ const description = document.createElement('p');
 const category = document.createElement('p');
 const price = document.createElement('p');
 const button = document.createElement('button');
-const numberSelector = document.createElement('select');
-const option1 = document.createElement('option');
-const option2 = document.createElement('option');
-const option3 = document.createElement('option');
+// const numberSelector = document.createElement('select');
```

Comment:
When you've got time, and get this to work, feel free to resubmit for more points.